deploy-board/deploy_board/templates/clusters/base_images.tmpl

```diff
@@ -27,11 +27,19 @@
                 {{ base_image.id }}
                 {% endif %}
             </td>
-            {% if base_image.tag %}
-            <td> <span class="glyphicon glyphicon-ok-sign"></span>Current Golden</td>
-            {% else %}
-            <td></td>
-            {% endif %}
+            <td>
+                {% if base_image.golden_latest %}
+                    <span class="label label-primary">GOLDEN_LATEST</span>
+                {% endif %}
+                {% if base_image.golden_canary %}
+                    <span class="label label-success">GOLDEN_CANARY</span>
+                {% endif %}
+                {% if base_image.tag %}
```

Comment:
so base_image.tag is used to tell if it's a current golden? can you update the name to reflect its current meaning. It might be confused later on.

Reply:
make sense, I can improve the name